docs/examples/training.ts

```diff
@@ -25,7 +25,8 @@ async function main (): Promise<void> {
   const NAME: string = 'titanic'
 
   // Launch a server instance
-  const [server, url] = await startServer()
+  const disco = await Server.of(defaultTasks.simpleFace, defaultTasks.titanic)
+  const [server, url] = await disco.serve()
```

Comment:
```suggestion
  const discoServer = await Server.of(defaultTasks.simpleFace, defaultTasks.titanic)
  const [server, url] = await discoServer.serve()
```
Same comment about the naming conflict with `runUser`